doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -5483,6 +5483,41 @@ Calling ``nrf_connect()`` immediately causes fail
 
   **Affected platforms:** nRF9160
 
+Known issues for custom boards
+******************************
+
+.. rst-class:: v2-8-0
+
+ VSC-2817: Custom boards for the nRF54L15 rev. Engineering B include Kconfig options for the rev. Engineering A
+    If you generated a custom board based on the nRF54L15 rev. Engineering B using the |NCS| v2.8.0, the custom board files mistakengly include the following Kconfig select options for the Engineering A version of the SoC:
+
+    * ``SOC_NRF54L15_ENGA_CPUAPP``
+    * ``SOC_NRF54L15_ENGA_CPUFLPR``
+
+    **Affected platforms:** nRF54L15 (rev. Engineering B)
+
+    **Workaround:** In the Kconfig file for your custom board (named :file:`Kconfig.<custom_board_name>`, where ``<custom_board_name>`` is the name of your board), manually replace the select options under the ``BOARD_CUSTOM_BOARD_NAME`` Kconfig option:
+
+    * ``SOC_NRF54L15_ENGA_CPUAPP`` with ``SOC_NRF54L15_CPUAPP``
+    * ``SOC_NRF54L15_ENGA_CPUFLPR`` with ``SOC_NRF54L15_CPUFLPR``
+
+    Then, :ref:`manually regenerate the board <zephyr:custom_board_definition>`.
+
+.. rst-class:: v2-9-0
+
+ VSC-2817: Custom boards for the nRF54L15 rev. Engineering B are missing :kconfig:option:`CONFIG_SOC_NRF_FORCE_CONSTLAT` setting
```

Comment:
This should be the opposite. The KConfig entry is to be removed not added

Reply:
See https://github.com/nrfconnect/vscode-nrf-connect/pull/4505

Reply:
Updated here: https://github.com/nrfconnect/sdk-nrf/compare/fe4ce68460f760c8923768c3fdeeace8adf4c48d..9ac92d6178d6f6bbd91b9d0e820c130f9be7ad57